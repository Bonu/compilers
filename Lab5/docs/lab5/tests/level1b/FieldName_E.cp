// Main: Test identifier references.

class Main{

   // int variables
    int i2 = main;                // f is not a field
    int i3 = OtherClass.f;     // OtherClass.f is not a field

  void main(){
  }
}

class OtherClass {

}
