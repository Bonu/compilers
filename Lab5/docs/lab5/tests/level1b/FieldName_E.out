line 6: the name `main' is not a field in this context
line 7: unable to resolve name -- OtherClass.f
2 errors were detected
